postgresql/README.md

```diff
@@ -1,10 +1,80 @@
 # PostgreSQL
 
+This packages PostgreSQL in a Habitat package that can be run with the Habitat supervisor.
```

Comment:
Style note: Supervisor, Builder, and Studio are all capitalized, everything else (depot, cli) are lowercase. :)